hugo/layouts/projects/single.html

```diff
@@ -5,6 +5,18 @@
 <body class="projects-page">
     {{ partial "header.html" $headerCtx }}
     <section class="projects-hero">
```

Comment:
It would be great if it would be more similar to the [home hero](https://github.com/src-d/landing/blob/master/hugo/layouts/partials/home/hero.html)
It would be great if it could be reused in some way the hero for the whole site (passing a template for the variable stuff)
In case it could not be reused at all, at least should have all the same markup structure.